action_plan Adecu_ActionPlanning_result

# trajectory planning

trajectory_plan Adecu_TrajPlanning_result

# vel planning

velocity_plan Adecu_VelPlanning_result
